exporter/collector/integrationtest/cmd/recordfixtures/main.go

```diff
@@ -48,18 +48,31 @@ func (t *FakeTesting) FailNow() {
 func (t *FakeTesting) Helper()      {}
 func (t *FakeTesting) Name() string { return "record fixtures" }
 
+type fixtureRecorder map[string]string
+
+func (fr fixtureRecorder) checkDuplicate(test testcases.TestCase) error {
+	if duplicate, ok := fr[test.ExpectFixturePath]; ok {
+		return fmt.Errorf("test %s writes to the same fixture %s as test %s", test.Name, test.ExpectFixturePath, duplicate)
+	}
+	fr[test.ExpectFixturePath] = test.Name
+	return nil
+}
+
 func main() {
 	ctx := context.Background()
 	endTime := time.Now()
 	startTime := endTime.Add(-time.Second)
 	t := &FakeTesting{}
 
-	recordLogs(ctx, t, endTime)
-	recordMetrics(ctx, t, startTime, endTime)
-	recordTraces(ctx, t, startTime, endTime)
+	// track the output fixtures to catch tests overwriting another test's fixtures
+	ft := make(fixtureRecorder)
```

Comment:
nit based on initials

```suggestion
	fr := make(fixtureRecorder)
```